lib/filesystem.js

```diff
@@ -81,7 +81,7 @@ class Filesystem {
         const stream = fs.createReadStream(p)
 
         stream.pipe(tr).pipe(out)
-        return tr.on('end', function () {
```

Comment:
I am wrong, "close" is correct, https://github.com/jprichardson/node-fs-extra/issues/326